Add plugin: Tasks Cleaner

lowitea opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/lowitea/obsidian-tasks-cleaner

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [x]  Linux
  - [x]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[# Obsidian Tasks Cleaner Plugin](https://github.com/lowitea/obsidian-tasks-cleaner/blob/8c6f966726366d3fbd676bd591fbf689913ea273/README.md#L1)
Naming something "Obsidian xyz" is reserved for first party products we create, please change this header.

["description": "Task Cleaner for completed tasks.",](https://github.com/lowitea/obsidian-tasks-cleaner/blob/8c6f966726366d3fbd676bd591fbf689913ea273/manifest.json#L6)
The description should not repeat the plugin name.

["author": "lowit <ea@lowit.ru>",](https://github.com/lowitea/obsidian-tasks-cleaner/blob/8c6f966726366d3fbd676bd591fbf689913ea273/manifest.json#L7)
This field is displayed in the sidebar and in the description for the plugin details page. Are you sure you want to have your email listed so prominently?
![image](https://user-images.githubusercontent.com/693981/213185753-7b7208d0-c614-4367-8f3b-b2ad4de02cad.png)

You should also add commands that activates the plugins functionality, the ribbon can be hidden and might be hidden by default in a future update.

[containerEl.createEl("h2", { text: "Tasks Cleaner Cleaner" });](https://github.com/lowitea/obsidian-tasks-cleaner/blob/8c6f966726366d3fbd676bd591fbf689913ea273/main.ts#L171)
[Don't add a top-level heading in the settings tab, such as "General", "Settings", or the name of your plugin.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)

lowitea commented:
@joethei thank you for this review!

> # Obsidian Tasks Cleaner Plugin

No problem. I have removed it.

> "description": "Task Cleaner for completed tasks.",

Fixed

> "author": "lowit ea@lowit.ru",
> This field is displayed in the sidebar and in the description for the plugin details page. Are you sure you want to have your email listed so prominently?

Of course not :) Fixed

> You should also add commands that activates the plugins functionality, the ribbon can be hidden and might be hidden by default in a future update.

Done

> containerEl.createEl("h2", { text: "Tasks Cleaner Cleaner" });

Done

All comments have been resolved :)
